sql/sys_vars.inl

```diff
@@ -2330,22 +2330,22 @@ bool update_multi_source_variable(sys_var *self,
 
 class Sys_var_multi_source_ulonglong :public Sys_var_ulonglong
 { 
-  ptrdiff_t master_info_offset;
+  mi_ulonglong_accessor_function mi_accessor_func;
   on_multi_source_update_function update_multi_source_variable_func;
 public:
   Sys_var_multi_source_ulonglong(const char *name_arg,
                              const char *comment, int flag_args,
                              ptrdiff_t off, size_t size,
                              CMD_LINE getopt,
-                             ptrdiff_t master_info_offset_arg,
+                             mi_ulonglong_accessor_function mi_accessor_arg,
                              size_t master_info_arg_size,
```

Comment:
Here we could remove the master_info_arg_size argument. It's only used to assert that the MASTER_INFO_VAR macro is not used with the wrong type. However when using the proper C++ pointer-to-member-function (or pointer-to-member), the code is type-safe and the compiler will catch any error, so the assertion is not needed.
Just a suggestion.